src/Microsoft.OData.Client/Materialization/ODataEntityMaterializer.cs

```diff
@@ -386,9 +386,13 @@ internal static ProjectionPlan CreatePlanForShallowMaterialization(Type lastSegm
         /// <param name="entry">Entry to get sub-entry from.</param>
         /// <param name="name">Name of sub-entry.</param>
         /// <param name="materializerContext">The materializer context.</param>
-        /// <returns>The sub-entry (never null).</returns>
+        /// <returns>The sub-entry (or null if materializerContext.AutoNullPropagation is true).</returns>

```

Comment:
Is it a breaking change?

Reply:
Default value of AutoNullPropagation is false - meaning it is not a breaking change.